index.js

```diff
@@ -3,4 +3,320 @@ function getGreeting (firstName, Surname) {
+const second = firstNumber;
+firstNumber = secondNumber;
+secondNumber = second;
+
+console.log(firstNumber);
+console.log(secondNumber);
+
+// Create variables that hold grades from three different classes:•
+// Physics•
+// Math•
+// English•
+// Calculate the average of the grades. If the average equals 4 out of 5 or higher, print "Good job!".
+//     Otherwise, print "Try harder!".
+
+const physicsGrade = 4;
+const mathGrade = 5;
+const englishGrade = 2;
+const averageGrade = (physicsGrade + mathGrade + englishGrade)/3
+
+if (averageGrade >= 4) {
+    console.log('Good job');
+} else {
+    console.log('Try harder')
+}
+
+// Write the convertMinutesToHours function.
+
+function convertMinutesToHours(minutes) {
+    return minutes / 60;
+}
+
+const hours = convertMinutesToHours(75);
+const fiveHours = convertMinutesToHours(300);
+const tenHours = convertMinutesToHours(600);
+console.log(hours);
+console.log(fiveHours);
+console.log(tenHours);
+
+// Write the isTextUppercase function.
+
+function isTextUppercase(text) {
+  if (text.toUpperCase() === text) {
+      return true;
+  } else {
+      return false;
+  }
+}
+
+const lowerCase = isTextUppercase('Hello'); // false
+const upperCase = isTextUppercase('HELLO'); // true
+
+console.log(lowerCase);
+console.log(upperCase);
+
+//Codewars - Create a function that checks if a number n is divisible by two numbers x AND y. All inputs are positive, non-zero numbers.
+function isDivisible(n, x, y) {
+    if (n % x === 0 && n % y === 0) {
+        return true;
+    } else {
+        return false;
+    }
+}
+const divisibleExampleOne = isDivisible(3,1,3);
+console.log(divisibleExampleOne);
+const divisibleExampleTwo = isDivisible(12,2,6);
+console.log(divisibleExampleTwo);
+const divisibleExampleThree = isDivisible(100,5,3);
+console.log(divisibleExampleThree);
+const divisibleExampleFour = isDivisible(12,7,5);
+console.log(divisibleExampleFour);
+
+
+// Write the getBillboardPrice function that accepts two arguments:•
+// the text intended to use on the billboard•
+// the cost of a single character
+function getBillboardPrice(text, cost) {
+    return (text + ' ' + cost * 12);
+}
+
+const billboard = getBillboardPrice('Hello world!', 10); // 120
+const billboardTwo = getBillboardPrice('Hello world!', 15); // 180
+const billboardThree = getBillboardPrice('To be, or not to be', 20); // 380
+console.log(billboard);
+console.log(billboardTwo);
+console.log(billboardThree);
+
+// given an integer or a floating-point number, find its opposite.
+
+function oppositeInteger(number) {
+    return (-number);
+}
+const numberOne = oppositeInteger(1);
+console.log(numberOne);
+const numberTwo = oppositeInteger(14);
+console.log(numberTwo);
+const numberThree = oppositeInteger(8);
+console.log(numberThree);
+
+// In this simple assignment you are given a number and have to make it negative. But maybe the number is already negative?
+function negativeInteger(num) {
+    if (num > 0) {
+        return (num * -1);
```

Comment:
The round brackets in `return (num * -1);` are unnecessary. Remove them, please.